samplers/samplers_test.go

```diff
@@ -8,8 +8,11 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stripe/veneur/ssf"
 )
 
+const ε = .01
```

Comment:
nit: i've [done this in the past](https://github.com/sdboyer/gogl/blob/master/rand/bernoulli.go#L25), and ended up regretting it, just because it meant copy/pasting the character. def not a blocker, just a perspective 😄